### PR TITLE
Generate static type

### PR DESCRIPTION
**Pull request by EPashkin**
Check parser changes

### CODE REVIEW

File: src/library.rs [hunk 1]
```diff
@@ -159,6 +159,7 @@ pub struct Interface {
 pub struct Class {
     pub name: String,
     pub glib_name: String,
+    pub get_type_func_name: String,
```
Comment (gkoz) [inline src/library.rs]: Hmm, maybe it's time to make these two names consistent with the xmls.

File: src/parser.rs [hunk 2]
```diff
@@ -165,11 +165,13 @@ impl Library {
         }
 
         let glib_name = attrs.get("type-name").unwrap_or(name);
+        let get_type_func_name = attrs.get("get-type").unwrap_or("None");
```
Comment (gkoz) [inline src/parser.rs]: This should use the same error handling as `"name"`:
```rust
let get_type_func_name = try!(
    attrs.get("get-type").ok_or_else(|| error!("Missing get-type attribute", parser)));
```
and located above the loop so the `parser` would report the correct position.

Comment (EPashkin) [inline src/parser.rs]: Ok, do it after #8 merged

File: src/library.rs (hunk 1 above)
Comment (EPashkin) [inline src/library.rs]: glib_type_name, glib_get_type ?

**Comment by EPashkin** [comment]
Fixed parser, change class names leave to you

**Pull request merged**